lab-nikko/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+
+if(process.argv[2]) var name = process.argv[2];
+if(process.argv[2] === undefined) var name = 'Carne Asuhdude';
```

Comment:
Yah ternary operators, we did that in 301! I'll take a stab at it, and come find you with any questions, thanks!